test/src/test/java/org/ethereum/beacon/test/SszTests.java

```diff
@@ -33,6 +33,8 @@ public void testSszStatic() {
         input -> {
           SszStaticRunner testRunner = new SszStaticRunner(input.getValue0(), input.getValue1());
           return testRunner.run();
-        });
+        },
+        getRecommendedThreadCount() // run it in parallel, a lot of tests
```

Comment:
IMO, it would be better to call `TestUtils.getTestExecutor(boolean parallel)` at lower layer with given `parallel` parameter.